website_snippet_image_gallery/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Website Snippet Image Gallery
+==================================
+
+Description
+-----------
+
+This module was written to add the functionality of image gallery in website with hover effects.
```

Comment:
This module adds the functionality of image gallery in website with hover effects.